### PR TITLE
[WIP][ACTP-310] Login Page - Fix HTML structure

### PR DESCRIPTION
**Pull request by AntonTsymuk**
Related to : https://oat-sa.atlassian.net/browse/ACTP-310
 
Update home link title and page title of login page
  
#### How to test
 
- prepare an instance of TAO with taoAct extension
- navigate to login page and check logo home link title and page title
- make sure to check with fresh installation and with update using updater script 
  
#### Dependencies

Companion PR:
 - [ ] https://github.com/oat-sa/extension-tao-act/pull/1306

### CODE REVIEW

File: config/default/login.conf.php [hunk 1]
```diff
@@ -27,5 +27,6 @@
      * @type bool
      */
     'disableAutocomplete' => false,
-    'enablePasswordReveal' => false
+    'enablePasswordReveal' => false,
+    'title' => 'TAO Login'
```
Comment (jsconan) [inline config/default/login.conf.php]: Unfortunately, this is not translatable. Could you use a service instead?

Comment (AntonTsymuk) [reply config/default/login.conf.php]: yes, but i will pass this value later into i18n helper to get translation inside the login controller

Comment (AntonTsymuk) [reply config/default/login.conf.php]: https://github.com/oat-sa/tao-core/pull/2402/files#diff-ce7f65685c6da98902fa941ad7f8e4b5R235

Comment (jsconan) [reply config/default/login.conf.php]: A dedicated service would avoid possible translation issues. Anyway, it would require a backend dev.

**Pull request closed without merging**